lightwood/analysis/explain.py

```diff
@@ -80,7 +81,13 @@ def explain(data: pd.DataFrame,
     # ------------------------- #
     # Additional Explanations
     # ------------------------- #
-    for block in explainer_blocks:
-        row_insights, global_insights = block.explain(insights, **{})
+    if len(explainer_blocks) > 0:
+        exec(lightwood.api.json_ai.IMPORTS_FOR_EXTERNAL_DIRS, globals())
+        exec(lightwood.api.json_ai.IMPORT_EXTERNAL_DIRS, globals())
+
+        for dirpath in explainer_blocks:
+            module, block_name = dirpath.split(".")
+            block = getattr(eval(module), block_name)()
+            row_insights, global_insights = block.explain(insights, **kwargs)
```

Comment:
Same comment as above, don't think this should be happening here.

Reply:
I'd be better for `explainer_blocks` to be a list of already existing objects that's e.g. instantiate when `class Predictor` is instantiated and inlined into the predictor code by `json_ai.py`